tests/acceptance/pageObjects/profilePage.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  commands: {
+    getUserProfileName: async function () {
+      let userInfo
+      await this
+        .waitForElementVisible('@profileInfoContainer')
+        .waitForElementVisible('@userProfileName')
+        .api.element('@userProfileName', result => {
+          this.api.elementIdText(result.value.ELEMENT, text => {
+            userInfo = text.value
+          })
+        })
+      return userInfo
+    },
+    browseToManageAccount: function () {
+      return this.waitForElementVisible('@manageAccount')
+        .click('@manageAccount')
+    }
+  },
+  elements: {
+    profileInfoContainer: {
+      selector: '//div[@class="uk-card-body uk-flex uk-flex-middle uk-flex-column"]',
```

Comment:
this is too generic, we will be in trouble with that xpath, other elements might have the same one

Reply:
fixed